src/inc/msquichelper.h

```diff
@@ -304,7 +304,7 @@ IsValue(
     _In_z_ const char* toTestAgainst
     )
 {
-    return _strnicmp(name, toTestAgainst, CXPLAT_MIN(strlen(name), strlen(toTestAgainst))) == 0;
+    return _strnicmp(name, toTestAgainst, strlen(toTestAgainst)) == 0;
```

Comment:
Why was this change made?

Reply:
We wanted to do an exact match, not just on the shortest part. This is because we're using some of the same prefix and we don't want incorrect matches.